stake-pool/py/stake_pool/instructions.py

```diff
@@ -543,6 +543,94 @@ class UpdateTokenMetadataParams(NamedTuple):
     """URI of the uploaded metadata of the spl-token."""
 
 
+class IncreaseAdditionalValidatorStakeParams(NamedTuple):
+    """(Staker only) Increase stake on a validator from the reserve account."""
+
+    # Accounts
+    program_id: PublicKey
+    """SPL Stake Pool program account."""
+    stake_pool: PublicKey
+    """`[]` Stake pool."""
+    staker: PublicKey
+    """`[s]` Staker."""
+    withdraw_authority: PublicKey
+    """`[]` Stake pool withdraw authority."""
+    validator_list: PublicKey
+    """`[w]` Validator stake list storage account."""
+    reserve_stake: PublicKey
+    """`[w]` Stake pool's reserve."""
+    transient_stake: PublicKey
+    """`[w]` Transient stake account to receive split."""
+    validator_stake: PublicKey
+    """`[]` Canonical stake account to check."""
+    validator_vote: PublicKey
+    """`[]` Validator vote account to delegate to."""
+    clock_sysvar: PublicKey
+    """`[]` Clock sysvar."""
+    rent_sysvar: PublicKey
+    """`[]` Rent sysvar."""
+    stake_history_sysvar: PublicKey
+    """'[]' Stake history sysvar."""
+    stake_config_sysvar: PublicKey
+    """'[]' Stake config sysvar."""
+    system_program_id: PublicKey
+    """`[]` System program."""
+    stake_program_id: PublicKey
+    """`[]` Stake program."""
+
+    # Params
+    lamports: int
+    """Amount of lamports to increase on the given validator."""
+    transient_stake_seed: int
+    """Seed to used to create the transient stake account."""
+    ephemeral_stake: PublicKey
+    """The ephemeral stake account used during the operation."""
```

Comment:
micro-nit: can you put this in the list with the rest of the `Accounts` at its correct location?